src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```
@@ -333,6 +333,14 @@
       <Output TaskParameter="Dependencies" ItemName="_BlazorResolvedAssemblyUnlinked" />
     </ResolveBlazorRuntimeDependencies>
 
+    <ItemGroup Condition="'$(BlazorWebAssembly18NAssemblies)' != 'none'">
+      <!--
+        Unless the user has asked for no-assemblies, copy all I18N assemblies to the build output.
+        We do not want to decipher the linker's format for passing in I18N options in our build targets
+      -->
+      <_BlazorResolvedAssemblyUnlinked Include="$(ComponentsWebAssemblyBaseClassLibraryPath)I18N*.dll" />
```

Is the following valid?
```suggestion
      <_BlazorResolvedAssemblyUnlinked Include="$(ComponentsWebAssemblyBaseClassLibraryPath)I18N.*.dll" />
```

---

There's a `I18N.dll` that is always required.